src/components/ec-alert/ec-alert.vue

```diff
@@ -153,54 +187,87 @@ export default {
     @apply tw-rounded;
     @apply tw-absolute tw-top-8 tw-right-8;
     @apply tw-cursor-pointer;
-    @apply tw-fill-gray-8;
 
     line-height: 0;
 
     &:focus {
       @apply tw-outline-none;
-      @apply tw-bg-gray-8;
     }
 
-    &:hover {
-      @apply tw-bg-gray-8;
+    &:hover,
```

Comment:
This block of code generates invalid CSS and doesn't work
![Screenshot from 2020-10-22 09-38-33](https://user-images.githubusercontent.com/5777410/96846907-6d5ed900-144a-11eb-81e6-d7f09ce65c27.png)

Reply:
You're right, I've just tested it, and doesn't apply anything, so we can remove it!